### PR TITLE
Allow linter warnings but be strict on rules

### PR DESCRIPTION
**Pull request by Half-Shot**
So we can have greater visibility on what needs fixing, rather than resorting to ignore comments.

### CODE REVIEW

File: .eslintrcts.json [hunk 1]
```diff
@@ -13,72 +13,73 @@
...
+    "no-spaced-func": "error",
+    "no-trailing-spaces": "error",
+    "keyword-spacing": ["error", {"before": true, "after": true}],
+    "space-before-blocks": ["error", "always"],
+    "@typescript-eslint/explicit-function-return-type": "off",
     "camelcase": ["error", { "properties": "never" }],
-    "@typescript-eslint/ban-ts-ignore": 0,
-    "no-unused-vars": 0,
+    "@typescript-eslint/ban-ts-ignore": "off",
+    "no-unused-vars": "off",
```
Comment (jaller94) [inline .eslintrcts.json:79]: 🤨

Comment (Half-Shot) [reply .eslintrcts.json:79]: ?

Comment (jaller94) [reply .eslintrcts.json:79]: I am wondering why unused variables need to be allowed for now, but I don't think it's a blocker.

Comment (Half-Shot) [reply .eslintrcts.json:79]: Oh, it was before. I just changed it to a more readable string.

**Pull request merged**